plugins/orchestrator-backend/src/service/OrchestratorService.ts

```diff
@@ -54,19 +54,13 @@ export class OrchestratorService {
   public async fetchInstances(args: {
     pagination?: Pagination;
   }): Promise<ProcessInstance[]> {
-    if (this.workflowCacheService.isEmpty()) {
-      return [];
-    }
```

Comment:
I 'd like to understand if there can be consequences in removing these sections

Reply:
The updated process works as follows:

We always call the APIs and send the list of IDs that are in the cache.
Before the fix, we didn't call the API if the cache was empty thus hiding the error in case the DataIndex fails.
After the fix, we now call the API even when the cache is empty, sending an empty list.
If the API call fails, an error will be displayed.